yarn.lock

```diff
@@ -19355,7 +19365,7 @@ __metadata:
     "@grafana/e2e": "workspace:*"
     "@grafana/e2e-selectors": "workspace:*"
     "@grafana/eslint-config": 2.5.1
-    "@grafana/experimental": canary
+    "@grafana/experimental": ^0.0.2-canary.6
```

Comment:
```suggestion
    "@grafana/experimental": latest
```

I guess that should resolve to the same package. Not really sure about the ideas of having canary and latest for this package.

Reply:
@mckn ?? the lock file I usually let yarn handle

Reply:
I ran `yarn up @grafana/experimental`, is there another command to run?

Reply:
ah, sorry - I commented on the wrong file 🙈

Reply:
or I read this PR totally wrong. I thought you had specified the version in the package.json. But then you can ignore this comment.